test/prototype_transforms_kernel_infos.py

```diff
@@ -668,8 +669,7 @@ def sample_inputs_affine_video():
 def sample_inputs_convert_format_bounding_box():
     formats = list(datapoints.BoundingBoxFormat)
     for bounding_box_loader, new_format in itertools.product(make_bounding_box_loaders(formats=formats), formats):
-        yield ArgsKwargs(bounding_box_loader, new_format=new_format)
-        yield ArgsKwargs(bounding_box_loader.unwrap(), old_format=bounding_box_loader.format, new_format=new_format)
+        yield ArgsKwargs(bounding_box_loader, old_format=bounding_box_loader.format, new_format=new_format)
```

Comment:
Don't we still want to test both pure-tensors *and* `BoundingBoxes`, at least for those "hybrid" kernels ? (perhaps that's still covered?)

Reply:
It wasn't until 99dd341, but now it is :innocent: So the kernel tests actually only test plain tensors, but the dispatcher tests take over the other part.